data/examples/README.md

```diff
@@ -1,9 +0,0 @@
-# The DFHack Example Configuration File Library
```

Comment:
Forgot to ask: did you intend to remove this? It does break the link at https://docs.dfhack.org/en/stable/docs/guides/examples-guide.html  - not sure off the top of my head where the files that page refers ended up.

Reply:
good catch. I did intend to remove the README since it's no longer the root of a sprawling hierarchy but rather just a single file, but it would not be so out of place just to move it to `dfhack-config/init/examples`. I'll take care of that (and the linked doc)